### PR TITLE
Add Python 3.9 to Travis tests

### PR DESCRIPTION
**Pull request by hosseinsh**
Adding Python 3.9 to tests, and dropping Python 3.7, since it is breaking some of our package updates that require a minimum of Python 3.8, and Postgres 9.4, as it reached EOL.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,20 +4,10 @@ node_js:
...
-        postgresql: "9.4"
-        chrome: stable
-      services:
-        - xvfb
-    - name: "python3.7-postgresql-10-bionic"
-      dist: bionic
-      language: python
-      python: "3.7"
+      python: "3.9"
       env: TOXENV=py37
```
Comment (jtschladen) [inline .travis.yml:11]: Should this be `TOXENV=py39`?

Comment (hosseinsh) [reply .travis.yml:11]: ahh, good catch thanks.

**Review by jtschladen — APPROVED** [review]

**Comment by jtschladen** [comment]
If you're trying to cover everything, the mean [README](https://github.com/Netflix/lemur/blob/master/README.rst) has a mention of 3.7.

File: .readthedocs.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ formats: all
 
 # Set the version of Python and requirements required to build the docs
 python:
-   version: 3.7
+   version: 3.9
```
Comment (jtschladen) [inline .readthedocs.yml:18]: Apparently not officially supported yet, unless we switch to the beta image:
```
Problem in your project's configuration. Invalid "python.version": expected one of (2, 2.7, 3, 3.5, 3.6, 3.7, 3.8, pypy3.5), got 3.9
```

https://github.com/readthedocs/readthedocs.org/issues/7554

Comment (hosseinsh) [reply .readthedocs.yml]: then 3.8 should do it, which is actually the min version needed for Bumping flake8 from 3.9.2 to 4.0.1

**Pull request merged**